Core/Importers/PrivatTransactionsImporter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using ExcelDataReader;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Core.Importers
+{
+    public class PrivatTransactionsImporter : TransactionsImporter
+    {
+        public override List<Transaction> Load(Stream file)
+        {
+            System.Threading.Thread.CurrentThread.CurrentCulture =
+                System.Globalization.CultureInfo.CreateSpecificCulture("en-GB");
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+            
+            //remove this
+            file = File.Open("statements.xls", FileMode.Open, FileAccess.Read);
+
+            using var reader = ExcelReaderFactory.CreateBinaryReader(file);
+            var dataSet = reader.AsDataSet();
+            var dataTable = dataSet.Tables[0];
+
+            var transactions = new List<Transaction>();
+            for (var i = 2; i < dataTable.Rows.Count; i++)
+            {                
+                var date = dataTable.Rows[i][0].ToString();
+                var category = dataTable.Rows[i][2].ToString(); 
+                var cardNumber = dataTable.Rows[i][3].ToString();
+                var description = dataTable.Rows[i][4].ToString();
+                var amount = dataTable.Rows[i][5].ToString();
+                //todo: read currency from file
```

Comment:
Codacy found an issue: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)